cards/common-partials/thumbsfeedback.hbs

```diff
@@ -1,23 +1,24 @@
-<div class="{{cardType}}-feedbackWrapper">
-  <div class="{{cardType}}-feedbackContent">
+{{#if directAnswerClass}}
+  {{assign 'className' directAnswerClass}}
```

Comment:
Instead of using the SDK's built in helper, can we make a version of it for the theme that adds the variable in the current scope, instead of the root scope? Adding variables to data.root affects the global scope which could bite us later. You can register a custom helper using `ANSWERS.registerHelper` (we already have a few of these). The last param of every handlebars helper is a special "HelperOptions" param, which is what the SDK's assign helper is mutating
typescript interface is here but they have an "any" type for HelperOptions.data which is not that helpful, but you can do console.logs to inspect the shape
https://github.com/handlebars-lang/handlebars.js/blob/6790c080c641ef2b44e663800e1794fae180977a/types/index.d.ts#L45

Reply:
alternatively, if you don't want to mess with that, you could make the feedback HTML an inline partial, and that partial with the logic for choosing params. We use inline partials a good bit, they look something like this

```hbs
{{#if some condition}}
  {{> inline-partial-name someParam=1 }}
{{else if something else}}
  {{> inline-partial-name someParam=2 }}
{{/if}}
{{#*inline 'inline-partial-name'}}
// blah
{{/inline}}
```

Reply:
That makes sense. I'll change it to use an inline partial